src/server.ts

```diff
@@ -18,6 +18,7 @@ import { errorHandler, requestHandler } from './server/middleware'
 import { lambdaHandler } from './server/aws_lambda'
 import { AsyncStore } from './server/async_store'
 
+const kHoneybadgerStore = Symbol.for("kHoneybadgerStore");
```

Comment:
Curious, is there any specific reason you use a `Symbol` for the key to the store contents?

Reply:
Thanks for elaborating!

> I think Symbols are not enumerable, so they won't show up in a for..in loop or similar

I just confirmed that this is the case. And this is a good reason to use a `Symbol` for this case.